src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -1579,16 +1641,20 @@ public int unlinkIdCertificates(Collection<String> certIds) {
             return 0;
         }
 
-        String query = "UPDATE Consumer c" +
-            " SET c.idCert = NULL, c.updated = :date" +
-            " WHERE c.idCert.id IN (:cert_ids)";
+        String jpql = """
+            UPDATE Consumer c
+            SET c.idCert = NULL, c.updated = :date
```

Comment:
For clarity, and to avoid confusion with the above comment: *this* date parameter is fine, as even though we're just passing in a fixed date defined as "now" within the method, doing so ensures that all of the updates will have the same updated timestamp even across partitions.

It doesn't really matter in practice, but it at least serves some tangible purpose here.